torch/testing/_internal/common_cuda.py

```diff
@@ -36,8 +36,7 @@
 SM100OrLater = LazyVal(lambda: torch.cuda.is_available() and torch.cuda.get_device_capability() >= (10, 0))
 SM120OrLater = LazyVal(lambda: torch.cuda.is_available() and torch.cuda.get_device_capability() >= (12, 0))
 
-IS_THOR = LazyVal(lambda: torch.cuda.is_available() and torch.cuda.get_device_capability()[0] == 10
-                  and torch.cuda.get_device_capability()[1] > 0)
+IS_THOR = LazyVal(lambda: torch.cuda.is_available() and torch.cuda.get_device_capability() == (11, 0))
```

Comment:
PyTorch still supports CUDA 12.6 and CUDA 12.8. Please add a condition for those as well. Something like this:

```suggestion
IS_THOR = LazyVal(lambda: torch.cuda.is_available() and
        ((torch.cuda.get_device_capability() == (11, 0) and int(torch.version.cuda[:2]) >= 13) or
        (torch.cuda.get_device_capability() == (10, 1) and int(torch.version.cuda[:2]) < 13))
)
```